### PR TITLE
add more og tags

### PR DESCRIPTION
**Pull request by mowolf**

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -21,11 +21,32 @@ export default {
...
+      {
+        hid: "og:title",
+        property: "og:title",
+        content: "Analyze your WhatsApp Chats",
+      },
+      {
+        hid: "og:description",
+        property: "og:description",
+        content:
+          "Learn who writes the most and many more! You can even generate a PDF from your chat!",
```
Comment (PKehnel) [inline nuxt.config.js:38]:
```suggestion
          "America's Most Popular WhatsApp Analyzer ✓ Now offering Group chats ✓ Reveal your friends character ✓ No Chat Data is sent to a Server. Get Started now!"
```

Isnt this our standard description?

**Review by PKehnel — CHANGES_REQUESTED** [review]
Description ? Apart from that good.

File: nuxt.config.js (hunk 1 above)
Comment (PKehnel) [reply nuxt.config.js:38]: Sebi meint keine meta tags mehr in die nuxt.Config

File: nuxt.config.js [hunk 2]
```diff
@@ -21,11 +21,32 @@ export default {
...
+      },
       {
         hid: "og:image",
         property: "og:image",
         content: "/sharePreview.png",
       },
+      {
```
Comment (fellnerse) [inline nuxt.config.js:45]: not needed, is set by nuxt already

**Pull request closed without merging**